tests/integration/test_TpfaMonoCoupling.py

```diff
@@ -77,23 +77,16 @@ def generate_grids(self, n, xmax, ymax, split):
 
         g1.compute_geometry()
         g2.compute_geometry()
-        grids = [g2, g1]
+        grids = [g1, g2]
```

Comment:
General comments on tests: I have not reviewed these in detail, let me know if there is something I should look at in particular. 

Generally speaking, I agree with your idea that this PR to some extent defines a new truth for some of these tests.